policyengine_us/variables/gov/states/ok/tax/income/credits/ok_child_care_child_tax_credit.py

```diff
@@ -26,6 +26,9 @@ def formula(tax_unit, period, parameters):
         ok_ctc = us_ctc * p.child.ctc_fraction
         # determine prorated fraction
         ok_agi = tax_unit("ok_agi", period)
-        prorate = min_(1, max_(0, where(us_agi != 0, ok_agi / us_agi, 0)))
+        agi_ratio = np.zeros_like(us_agi)
```

Comment:
```suggestion
        # Compute OK AGI as a share of US AGI.
        # Use a mask rather than where to avoid a divide-by-zero warning.
        agi_ratio = np.zeros_like(us_agi)
```